src/domains/fdr-app/00_monitor.tf

```diff
@@ -42,6 +42,6 @@ resource "azurerm_portal_dashboard" "fdr-dashboard" {
   })
 
   lifecycle {
-    ignore_changes = [dashboard_properties,tags]
+    ignore_changes = [dashboard_properties, tags]
```

Comment:
@cap-ang remove this `ignore_changes` and update `./dashboard/dashboard-apim-fdr.tpl" this trick its done during PROD release because [dash export](https://portal.azure.com/?l=en.en-us#@pagopait.onmicrosoft.com/dashboard/arm/subscriptions/b9fc9419-6097-45fe-9f74-ba0641c91912/resourcegroups/pagopa-p-monitor-rg/providers/microsoft.portal/dashboards/flussidirendicontazione-prod-fdr) from Az portal doesn't work properly